lib/smart_answer_flows/calculate-state-pension/calculate_state_pension.govspeak.erb

```diff
@@ -7,15 +7,18 @@
 <% end %>
 
 <% content_for :body do %>
-  Calculate when you’ll reach State Pension age or Pension Credit qualifying age and how much you may get in today’s money for your basic State Pension.
+  Calculate when you’ll reach State Pension age, Pension Credit qualifying age or fine out when you'll be eligible for free bus travel.
```

Comment:
Yes, I just realised that.

On 5 October 2015 at 14:26, Isabell Long <notifications@github.com> wrote:

> In
> lib/smart_answer_flows/calculate-state-pension/calculate_state_pension.govspeak.erb
> <https://github.com/alphagov/smart-answers/pull/1989#discussion_r41142642>
> :
>
> > @@ -7,15 +7,18 @@
> >  <% end %>
> >
> >  <% content_for :body do %>
> > -  Calculate when you’ll reach State Pension age or Pension Credit qualifying age and how much you may get in today’s money for your basic State Pension.
> > +  Calculate when you’ll reach State Pension age, Pension Credit qualifying age or fine out when you'll be eligible for free bus travel.
>
> Should "fine" be "find" here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/smart-answers/pull/1989/files#r41142642>.
>



-- 
[image: Cabinet Office - Government Digital Service]

*Liz Lutgendorff*
Content Analyst
Government Digital Service
Aviation House
London WC2B 6NH